nsv13/code/modules/overmap/fighters/_fighters.dm

```diff
@@ -532,54 +536,20 @@ Been a mess since 2018, we'll fix it someday (probably)
 	//check_overmap_elegibility(TRUE)
 
 /obj/structure/overmap/fighter/proc/throw_pilot() //Used when yeeting a pilot out of an exploding ship
+	var/m_damage = 200
```

Comment:
You might want to make this part of the proc as `throw_pilot(damage = 200)` so we can modify it if we want to use it for less serious accidents. (or make it not kill the pilot)